src/js/actions/menu.js

```diff
@@ -96,17 +96,22 @@ define(function (require, exports) {
                 return ps.performMenuCommand(payload);
             })
             .then(function (success) {
-                // FIXME: After the M2 release, remove the debug conjunct
                 if (global.debug && !success) {
-                    throw new Error("Menu command not available: " + payload.commandID);
+                    log.error("Menu command not available: " + payload.commandID);
```

Comment:
No all failed menu command are bad. For example, it should be acceptable for `clearPath` command to fail. So I replace the `throw` with `log.error` here.